templates/quarkus-camel-api-consumer/skeleton/pom.xml

```diff
@@ -33,6 +33,7 @@
         <maven-jar-plugin.version>3.4.2</maven-jar-plugin.version>
         <maven-resources-plugin.version>3.3.1</maven-resources-plugin.version>
         <maven-surefire-plugin.version>3.4.0</maven-surefire-plugin.version>
+        <registry.version>2.6.4.Final</registry.version>
```

Comment:
Because the only usage of this property is inside an if-statement, it would seem to make sense to only include this property when the same conditions match.